swank.lisp

```diff
@@ -2495,7 +2495,10 @@ Record compiler notes signalled as `compiler-condition's."
 (defslimefun compile-string-for-emacs (string buffer position filename policy)
   "Compile STRING (exerpted from BUFFER at POSITION).
 Record compiler notes signalled as `compiler-condition's."
-  (let ((offset (cadr (assoc :position position))))
+  (let* ((offset (cadr (assoc :position position)))
+         (line-column (assoc :line position))
+         (line (cadr line-column))
+         (column (caddr line-column)))
```

Comment:
Maybe turn this into:

```
   (line-column (cdr (assoc :line position)))
   (line (first line-column))
   (column (second line-column)))
```

if you agree that's more readable. Not a big deal.

Reply:
That looks better - I'll follow your suggestion.